server/src/pages/shotindex/view.js

```diff
@@ -463,6 +468,16 @@ class Card extends React.Component {
     this.downloadButton.blur();
     sendEvent("download", "myshots-tile");
   }
+
+  onClickFavorite(shot) {
+    if (shot.isFavorite) {
```

Comment:
`isFavorite` is set on each shot after it's instantiated from the JSON data and updated in `toggleFavoriteShot`.  Could consider changing it to a function in shot that returns a bool based on expiration, similar to what was done for `isOwner`.